src/ebonite/ext/flask/prebuild/Dockerfile.j2

```diff
@@ -0,0 +1,25 @@
+FROM python:{{ python_version }}-slim
+
+WORKDIR /app
+
+RUN true \
+    && apt update \
+    && apt install -y \
+        bash \
+        nginx \
+        supervisor \
+        gcc \
+    && pip install uwsgi==2.0.18 \
+    && apt remove gcc -y \
+    && apt autoremove -y \
+    && apt clean
+
+RUN pip install flask==1.1.2 flasgger==0.9.3
```

Comment:
maybe template it (at least flask) and use the current version in your system (to be consistent with things like run_model_server)

Reply:
My point was that we should not require user to install server packages in their Jupyter notebooks. Moreover, it is simpler to control when versions are specified by us and not by user environment. For ML libraries it is unrealistic (it could break user's models) but for servers it looks feasible and useful (server-related code is written by us and could be on the contrary broken by user-specified version).

Reply:
As for run_model_server I think it's OK not to be 100% consistent with it as it is a bit hackish by itself. For me it's more like a workaround to overcome slowness of images building (which was greatly reduced by this PR but we should work on it further).